iceoryx_utils/include/iceoryx_utils/internal/units/duration.hpp

```diff
@@ -233,35 +233,35 @@ class Duration
     /// @brief returns the duration in nanoseconds
     /// @note If the duration in nanoseconds is larger than an uint64_t can represent, it will be clamped to the
     /// uint64_t max value.
-    constexpr uint64_t nanoSeconds() const noexcept;
+    constexpr uint64_t toNanoSeconds() const noexcept;
```

Comment:
could you please make the `S` in `toNanoSeconds` lower case. Same with `toMicroSeconds` and `toMilliSeconds`

Reply:
I prefer on the other hand exactly this style `toNanoSeconds`! Because in my personal opinion it is easier to read - but I do not insist on it.

Reply:
but it's wrong in english :laughing: 
nanoseconds is one word

Reply:
I'll left this for tomorrow, so you guys can decide :D

Reply:
@enkeyz Thank you for your patience. 

@elBoberido  I think we need a duel. Could you please imagine how I slapped your face with my gauntlet. You as the challenged can choose your weapon I choose place and time and then let's rumble :D ...
I would suggest the following: We both eat one bite (but at least 10g) of https://en.wikipedia.org/wiki/Carolina_Reaper , the first one who gives up is out.

With the following rules:
1. The challenged has to start.
2. After one has taken a bite the other one has to take a bite after at most 60 seconds.
3. Multiple bites at once do count as one. It has to be alternating.
4. The duel has to be supervised by our slap bet commissioner @FerdinandSpitzschnueffler

Reply:
I guess @dkroenke has to break the tie

Reply:
Tiebreaker: The lowercase "s" make the deal because it's a common word and std::chrono also sticks to that. So i see no reason to differ from that. Sorry elfi...
Generally the iceoryx_utils extend the STL and should try to follow their conventions as good as possible. 
And this reminds me of that we should setup the Clang-Tidy...

Reply:
Okey, then it's decided. Made my final commit. :)